tflite2onnx/op/squared_difference.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import tflite
+import numpy as np
+
+from tflite2onnx import mapping
+from tflite2onnx.op.common import Operator
+from tflite2onnx.op.binary import Binary
+
+logger = logging.getLogger('tflite2onnx')
+
+
+# wrapper is used here to override Binary.type property
+class PowerWrapper(Binary):
+    @property
+    def type(self):
+        return 'Pow'
+
+
+class SquaredDifference(Operator):
+    # use subtraction as input operator and propagate output to power
+    TypeMapping = {
+        tflite.BuiltinOperator.SQUARED_DIFFERENCE: 'Sub',
```

Comment:
Maybe we need to fix this in case it confuses people that a TFLite SquaredDifference is implemented by an ONNX Sub. Let me take this item.